File: leetcode/387 - First Unique Character in a String/Solution.java
class Solution {
  public int firstUniqChar(String s) {
    for (int i = 0; i < s.length(); i++) {
      char c = s.charAt(i);
      int firstIndex = s.indexOf(c);
      int lastIndex = s.lastIndexOf(c);
      if (firstIndex == lastIndex)
        return firstIndex;
    }

    return -1;
  }
}

File: leetcode/387 - First Unique Character in a String/Solution.js
/**
 * @param {string} s
 * @return {number}
 */
var firstUniqChar = function (s) {
  for (let i = 0; i < s.length; i++) {
    let firstIndex = s.indexOf(s[i]);
    let lastIndex = s.lastIndexOf(s[i]);
    if (firstIndex == lastIndex) return firstIndex;
  }

  return -1;
};
